WebDriverAgentLib/Utilities/FBFailureProofTestCase.m

```diff
@@ -23,7 +23,15 @@ - (void)setUp
 {
   [super setUp];
   self.continueAfterFailure = YES;
-  self.internalImplementation = (_XCTestCaseImplementation *)[FBXCTestCaseImplementationFailureHoldingProxy proxyWithXCTestCaseImplementation:self.internalImplementation];
+  if ([self respondsToSelector:@selector(internalImplementation)]) {
```

Comment:
How about leave a comment Xcode 11.4 no longer have this API in L17 in PrivateHeaders/XCTest/XCTestCase.h ?

Reply:
added

Reply:
I updated the codes but the error still exists..... I'm using Xcode 11.4, IOS 11.1. In the WDA file I downloaded, the file FBFailureProofTestCaseTest.m was under path WebDriverAgentTests/IntegrationTests, not WebDriverAgentLib/Utilities. Did I download the wrong file?

Reply:
This file was `WebDriverAgentLib/Utilities/FBFailureProofTestCase.m`, not `FBFailureProofTestCaseTest.m`.
Did you find another issue in `FBFailureProofTestCaseTest.m`?

Reply:
No. It works fine now. Sorry. That was my stupid mistake. Thanks.